css/enhancedsteam.css

```
@@ -2347,6 +2330,7 @@ input[type=checkbox].es_dlc_selection:checked + label {
   font-size: 25px;
   text-align: center;
   float: left;
+  padding: 0 !important;
```

```suggestion
  padding: 0;
```
This selector is very specific, you don't need `!important`